src/main/java/io/arlas/server/app/ArlasServerConfiguration.java

```diff
@@ -119,7 +123,13 @@ public void check() throws ArlasConfigurationException {
         if (arlasServiceCSWEnabled == null) {
             arlasServiceCSWEnabled = false;
         }
-        if (collectionAutoDiscoverConfiguration == null) {
+        if(arlasServiceTagEnabled==null){
+            arlasServiceTagEnabled=false;
+        }
+        if(arlasServiceTagEnabled==null){
```

Comment:
Remove this `if` block to avoid duplicate code.

Reply:
ok